appsettings.json

```diff
@@ -6,12 +6,12 @@
     }
   },
   "AllowedHosts": "*", 
-  "GoogleAPI":{
+  "GoogleApi":{
     "AuthBaseUrl":"https://accounts.google.com/o/oauth2/v2/auth",
     "RequestTokenUrl":"https://oauth2.googleapis.com/token",
-    "ClientID": "204950401532-480uaukb9rm6dimheflms2s5d1v9r0s0.apps.googleusercontent.com",
-    "RedirectUrl":"http%3A//localhost:8080",
-    "Scope":"https%3A//www.googleapis.com/auth/calendar email openid", 
+    "ClientID": "",

```

Comment:
I'd leave this empty or add a dummy message to use another configuration that isn't tracked in git.